src/commands/run-tests.yml

```diff
@@ -14,6 +14,12 @@ parameters:
       folders using a colon-separated or semicolon-separated list.
     type: string
     default: ''
+  select-by-name:
+    description: >
+      Name or pattern used to select test suite elements. To create a test suite, MATLAB uses only the test elements
+      that match the specified name or pattern.
```

Comment:
Is this parameter an extension of https://www.mathworks.com/help/matlab/ref/matlab.unittest.testsuite.fromname.html#mw_54835d2c-0ac1-492f-b283-ad55aec0f547? Which wildcard characters can the pattern have? I think this is important information to include.

Reply:
I have added information about the wildcard characters. Additionally, the parameter uses name-value arguments from the testsuite function, as described below: 
 [https://www.mathworks.com/help/matlab/ref/testsuite.html#d126e1674941:~:text=C%3A%5Cwork%5CfolderB%22%5D-,Name%20%E2%80%94%20Name%20of%20test,-string%20array%20%7C](https://www.mathworks.com/help/matlab/ref/testsuite.html#d126e1674941:~:text=C%3A%5Cwork%5CfolderB%22%5D-,Name%20%E2%80%94%20Name%20of%20test,-string%20array%20%7C)

Reply:
Sounds good. Thank you, Vandit!